src/main/java/frc/robot/subsystems/Shooter.java

```diff
@@ -20,8 +23,11 @@
  * or X for shoot all. Does it store any state? No. Sensors? No.
  */
 
-public class Shooter extends SubsystemBase {
+public class Shooter extends SubsystemBase implements Loggable {
```

Comment:
I'm not a big fan of it, but we probably need to add a member variable for the calculated velocity and RPM so we can log those. They should be set in the appropriate functions below.